src/Evolution/Executables/ScalarTensor/ScalarTensorBase.hpp

```diff
@@ -397,24 +399,23 @@ struct ScalarTensorTemplateBase {
               volume_dim>>,
       Actions::MutateApply<gh::gauges::SetPiAndPhiFromConstraints<
           gh::ScalarTensor::AnalyticData::all_analytic_data, volume_dim>>,
-      Initialization::Actions::AddSimpleTags<
-          CurvedScalarWave::Initialization::InitializeConstraintDampingGammas<
-              volume_dim>>,
       Parallel::Actions::TerminatePhase>;
 
   // A tmpl::list of tags to be added to the GlobalCache by the
   // metavariables
-  using const_global_cache_tags =
-      tmpl::list<gh::gauges::Tags::GaugeCondition,
-                 evolution::initial_data::Tags::InitialData,
-                 gh::ConstraintDamping::Tags::DampingFunctionGamma0<
-                     volume_dim, Frame::Grid>,
-                 gh::ConstraintDamping::Tags::DampingFunctionGamma1<
-                     volume_dim, Frame::Grid>,
-                 gh::ConstraintDamping::Tags::DampingFunctionGamma2<
-                     volume_dim, Frame::Grid>,
-                 // Source parameters
-                 ScalarTensor::Tags::ScalarMass>;
+  using const_global_cache_tags = tmpl::list<
+      gh::gauges::Tags::GaugeCondition,
+      evolution::initial_data::Tags::InitialData,
+      gh::ConstraintDamping::Tags::DampingFunctionGamma0<volume_dim,
+                                                         Frame::Grid>,
+      gh::ConstraintDamping::Tags::DampingFunctionGamma1<volume_dim,
+                                                         Frame::Grid>,
+      gh::ConstraintDamping::Tags::DampingFunctionGamma2<volume_dim,
+                                                         Frame::Grid>,
```

Comment:
You'll catch this when you compile, but I think these tags got moved out of the `ConstraintDamping` namespace :)